docs/client.rst

```diff
@@ -187,401 +163,79 @@ are using Microsoft Visual Studio, you should do the following::
   > cmake -G"Visual Studio 16 2019" -DCMAKE_BUILD_TYPE=Release
   > MSBuild.exe trtis-clients.vcxproj -p:Configuration=Release
 
-When the build completes the libraries and examples can be found in
-trtis-clients/install. The C++ client examples will not be generated
-as those examples have not yet been ported to Windows. However, you
-can use the Python examples to test if the build is successful. To use
-the Python examples, you need to install the tensorrtserver Python package
-and other packages required by the examples::
-
-  > pip install trtis-clients/install/python/tensorrtserver-*.whl numpy pillow
+When the build completes the libraries can be found in
+trtis-clients/install/lib, the corresponding headers in
+trtis-clients/install/include, and the Python wheel file in
+trtis-clients/install/python. The trtis-clients/install directory will
```

Comment:
Should change the path in line 155 to use "/" for consistency, or the other way around (Windows seems to accept both notations nowadays)